controllers/events_controller.js

```diff
@@ -1106,3 +1231,36 @@ module.exports.generateCerti = async (req, res) => {
 		sendError(res, "Not eligible for certificate", BAD_REQUEST);
 	}
 };
+
+module.exports.sendEventMails = async (req, res) => {
+	let { type, users, event, subject, content } = req.body;
+	// type: event-reminder, event-followup, event-thanks
+	// users: [{ name, email }]
+	// event: event id
+
+	let events;
+	if (!subject && !content) {
+		events = await Event.findById(event);
```

Comment:
DB will return only one event here, can change the variable from plural to singular to avoid confusion later.

Reply:
events -> evnt